tools/add_triage_signature.py

```diff
@@ -582,14 +564,82 @@ def isVMware(host):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class AgentStepFailureSignature(Signature):
+    """
+    This signature creates a report of all agent step failures that were encountered in the logs
+    """
+    PATTERN = re.compile(
```

Comment:
It looks less worse with syntax highlighting:
![image](https://user-images.githubusercontent.com/10882062/113298345-63e47180-9304-11eb-87d1-2354c6af9259.png)

Reply:
yes, I'm not sure how to make it more readable
maybe separate it into a couple of regexps, as time, level and msg belong to the top-level message and msg is composed by several fields